app/qml/components/MMAmPmSwitch.qml

```diff
@@ -0,0 +1,55 @@
+/***************************************************************************
```

Comment:
wondering if not to put all "sub" components to subfolder (`components/calendar`) and just keep the calendar that can be used from rest of the application in `comonents` folder?